ios/keyman/Keyman/Keyman/AppDelegate.swift

```diff
@@ -55,9 +55,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
                    didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]? = nil) -> Bool {
     SentryManager.start()
 
-    let userDefaults = AppDelegate.activeUserDefaults()
-    SentryManager.enabled = userDefaults.bool(forKey: shouldReportErrorsKey)
-
```

Comment:
While this was necessary in the first pass of the PR, the rework that centralized the 'enabled' flag as part of `SentryManager` had some really interesting interactions with this.

At this stage, leaving these two lines in actually _breaks_ the error-reporting toggle behavior - it'll always be "enabled" for the system keyboard if left in, and I'm not clear on why.

There's a strong chance this is related to [this comment chain](https://github.com/keymanapp/keyman/pull/4106#discussion_r541986386).  It's a similar kind of behavior to what I saw there and feels pretty consistent with it.  I'm also not clear as to _why_ these lines cause an error here, as very similar lines are called successfully later (via `SentryManager.enabled`'s getter).

It'll take more investigation to figure out _exactly_ what's going on, but this feels like a strong lead.

Reply:
For why these lines causing a problem isn't clear:

`SentryManager.enabled`'s getter draws directly from `userDefaults.bool(forKey: shouldReportErrorsKey)` and sets to the same place.  The second line should basically be an "identity" / redundant statement.  There's a shot it was merely b/c the app-group identifier hadn't yet been set properly... but that's merely "a shot".

An interesting aspect of the error that had been occurring - the toggle's value would persist (solely) within the app until it was quit.  Relaunching the app would auto-reset the toggle for error reporting to "enabled"... as would re-launching the keyboard app extension.  Somehow, killing these two lines prevented both resets-to-enabled bugs.

So... how might this be a lead?  It's kind of conjecture for now, but... if, somehow, improperly-timed `set`s to `userDefaults` seem to retrieve and/or use unexpected values... perhaps a behavior might seem to preserve the original state of the toggle instead of the newly-select one in the right context?